src/networking/backend.ts

```diff
@@ -72,17 +74,17 @@ export class Backend {
     );
   }
 
-  async postPurchase(
+  async postCheckoutStart(
     appUserId: string,
     productId: string,
-    email: string,
     presentedOfferingContext: PresentedOfferingContext,
     purchaseOption: PurchaseOption,
-  ): Promise<PurchaseResponse> {
-    type PurchaseRequestBody = {
+    email?: string,
```

Comment:
`email` field is no longer required to initialize a purchase. This will allow us to move that field to the `state-needs-payment-info` and remove the `state-needs-auth-info`

Reply:
merge-conflict incoming!